targets/TARGET_NORDIC/TARGET_NRF5_SDK13/trng_api.c

```diff
@@ -42,29 +42,47 @@
 
 void trng_init(trng_t *obj)
 {
+    (void) obj;
+
     (void)nrf_drv_rng_init(NULL);
```

Comment:
NULL value passed in config pointer causes that rng peripheral will be configured without bias correction. Maybe we should consider to enable it to be sure about statistically uniform distribution of the random values? Of course enabling this function causes slow down generation of random numbers.